src/core/software/pupil-player.md

```diff
@@ -403,7 +403,16 @@ The `World Video Exporter` is loaded by default.
   <img src="../../media/core/imgs/export.jpg" style="display:flex;margin:0 auto;">
 </div>
 
-The export saves the world video as shown in Player, including all currently active visualizations (see the [Visualization Plugins](#visualization-plugins "Pupil Player visualization plugins documentation") section).
+The export saves the world video as shown in Player, including all currently active visualizations (see the [Visualization Plugins](#visualization-plugins "Pupil Player visualization plugins documentation") section). 
+
+Gereral overview for exported video files:
+
+The `world video exporter` exports the scene video file, together with numpy and .csv files containing timestamps corresponding to each frame. Data made available in world_timestamps.csv is timestamps in seconds from ..., and 'presentation timestamp' `pts`. This stands for presentation timestamp. It refers to the media file's internal time representation and can be used to seek to/identify specific frames within the media file.
```

Comment:
```suggestion
The World and Eye video exporter save the scene video file, together with [numpy](https://numpy.org/devdocs/reference/generated/numpy.lib.format.html) and csv files containing timestamps corresponding to each frame. 
The timestamps follow the [Pupil Time](/core/terminology/#timestamps) convention. The csv timestamp files include an additional `pts` column. This stands for presentation timestamp and refers to the media file's internal time representation. It can be used to seek to/identify specific frames within the media file.
```